datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -5674,6 +5674,11 @@ type DataJob implements EntityWithRelationships & Entity & BrowsableEntity {
     Whether or not this entity exists on DataHub
     """
     exists: Boolean
+
+    """
+    fine grained lineage
+    """
+    fineGrainedLineages: [FineGrainedLineage!]
```

Comment:
question below. typically we name the top level field in unison with the aspect, or at least structure in that way, in the case that we need to extend to other things. what do you think about doing that here?

Reply:
yup! makes sense